RELEASE_NOTES.md

```diff
@@ -2,6 +2,20 @@
 
 This section contains past updates we've sent to customers via Discord.
 
+## 0.8.0 (June 30, 2023)
+
+We've just released Shorebird v0.8.0 🎉
+
+- 🪟 windows installer fixes/improvements
```

Comment:
You could announce the free tier in this one (even though it technically went out in the last one) if you like. :)

Reply:
yeah good call